### DISCUSSION TITLE
LCD Backlight auto off

### DISCUSSION THREAD

**Discussion by cfi700** [ArminJo/JK-BMSToPylontechCAN#6] [Ideas]
It would be nice if the backlight of the display would turn off automatically after some time (e.g. 30sec).
When pressing the button with deactivated backlight, the "Big Info Page" could then be displayed again as the start page.

Best regards,
christian

**Comment by ArminJo** [comment]
Still under construction 😀

**Reply by ArminJo** [reply]
Please test it.

**Reply by cfi700** [reply]
Works perfectly. 
Had a few times activated the display and it has become dark again after some time as desired.
